images/ubuntu-slim/Makefile

```diff
@@ -1,22 +1,79 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 all: push
 
-TAG ?= 0.8
-PREFIX ?= gcr.io/google-containers/ubuntu-slim
+TAG ?= 0.9
+REGISTRY = gcr.io/google_containers
+ARCH ?= $(shell go env GOARCH)
+QEMUVERSION=v2.7.0
+
+IMGNAME = ubuntu-slim
+IMAGE = $(REGISTRY)/$(IMGNAME)
+MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
+
 BUILD_IMAGE ?= ubuntu-build
 TAR_FILE ?= rootfs.tar
 PUSH_TOOL ?= gcloud
 
+# Set default base image dynamically for each arch
+ifeq ($(ARCH),amd64)
+	BASEIMAGE?=ubuntu:16.04
+endif
+ifeq ($(ARCH),ppc64le)
```

Comment:
I'm able to find out images for other archs and here is the complete list, let me know if any change:

```
ubuntu:16.04
armhf/ubuntu:16.04
aarch64/ubuntu:16.04
ppc64le/ubuntu:16.04
s390x/ubuntu:16.04
```